napari_aicssegmentation/controller/batch_processing_controller.py

```diff
@@ -13,8 +14,56 @@ def __init__(self, application: IApplication, workflow_engine: WorkflowEngine):
         self._workflow_engine = workflow_engine
         self._view = BatchProcessingView(self)
 
+        #Should these go into a model?
```

Comment:
Technically yes, but since (for now) the controller won't have any values to update/pass back to the view, we don't really need a model. You could just have these parameters passed in the `run_batch` method for now and I think it'll be good enough. 
If in the future there are any interactions that require keeping / updated a state on this widget, we'll add a model.